CHANGES.md

```diff
@@ -1,3 +1,7 @@
+Airbase 88
+
+* Replace dependency-versions-check with Enforcer requireUpperBoundDeps
```

Comment:
Does it mean that i can now have a  dep twice+ (directly or indirectly) with different version numbers?